static/src/stylesheets/base/_inline-svgs.scss

```diff
@@ -6,6 +6,15 @@
     }
 }
 
+.inline-icon.light-grey {
+    fill: colour(neutral-3);
+}
+
+.inline-close.small {
```

Comment:
@NataliaLKB If you've not seen it already http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ is good, as the naming conventions there are the closest to the ones used on theguardian.com